test/test_twap.py

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/env python
+
+from os.path import join, realpath
+from hummingbot.strategy.market_trading_pair_tuple import MarketTradingPairTuple
+from decimal import Decimal
+import logging; logging.basicConfig(level=logging.ERROR)
+import pandas as pd
+from typing import List
+import unittest
+from hummingsim.backtest.backtest_market import BacktestMarket
+from hummingsim.backtest.market import (
+    AssetType,
+    Market,
+    MarketConfig,
+    QuantizationParams
+)
+from hummingsim.backtest.mock_order_book_loader import MockOrderBookLoader
+from hummingbot.core.clock import (
+    Clock,
+    ClockMode
+)
+from hummingbot.core.event.event_logger import EventLogger
+from hummingbot.core.event.events import (
+    MarketEvent,
+    OrderCancelledEvent,
+    TradeType,
+    OrderType,
+    OrderFilledEvent,
+    BuyOrderCompletedEvent,
+    SellOrderCompletedEvent,
+    TradeFee
+)
+from hummingbot.core.data_type.limit_order import LimitOrder
+from hummingbot.strategy.execution4 import Execution4TradeStrategy
```

Comment:
You need to update the strategy ref in your test case as well... otherwise it won't pass CI.